optuna_integration/sklearn/sklearn.py

```diff
@@ -280,7 +280,10 @@ def _cross_validate_with_pruning(
         if is_classifier(estimator):
             partial_fit_params = self.fit_params.copy()
             y = self.y.values if isinstance(self.y, pd.Series) else self.y
-            classes = np.unique(y)
+            if y is not None:
+                classes = np.unique(y)
+            else:
+                classes = np.array([None])
```

Comment:
Notes: `np.array([None])` seems to be a bit strange, but it is a return value of `np.array(None)`.